redisinsight/ui/src/pages/home/components/add-database-screen/components/connectivity-options/styles.module.scss

```diff
@@ -2,29 +2,72 @@
   font-weight: 400 !important;
   display: flex;
   align-items: center;
+}
 
-  .cloudIcon {
-    margin-right: 8px;
+.cloudSection {
+  .typeBtn {
+    min-width: 160px !important;
   }
 }
 
 .typeBtn {
+  position: relative;
   width: 100%;
-  height: 76px !important;
+  height: 84px !important;
+  padding: 0 12px;
 
   border-color: var(--separatorColorLight) !important;
   color: var(--buttonSecondaryTextColor) !important;
   box-shadow: none !important;
 
-  &.primary {
-    border-color: var(--euiColorSecondary) !important;
+  &.small {
+    padding: 0 8px;
+    height: 52px !important;
+
+    :global(.euiButton__text) {
+      flex-direction: row;
+    }
+
+    .btnIcon {
+      margin-right: 8px;
+    }
   }
 
   &:hover {
     background-color: transparent !important;
   }
-}
 
-.star {
-  margin-right: 4px;
+  :global(.euiButton__text) {
```

Comment:
Question: why do we need `:global`?

Reply:
since there is a global class, not scoped